dev-libs/soci/metadata.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>WolfgangESanyer@Gmail.com</email>
+		<name>Wolfgang E. Sanyer</name>
+	</maintainer>
+</pkgmetadata>
```

Comment:
You'll need a project or a person with `@gentoo.org` address to proxy commits for you.

Reply:
Where do i solicit such a thing?

Reply:
[Add](https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide#Proxied_maintainer_in_metadata.xml) proxy-maint as a project